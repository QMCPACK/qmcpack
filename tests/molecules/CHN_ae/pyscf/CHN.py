#! /usr/bin/env python3

from pyscf import gto
from pyscf import scf, dft, df

mol = gto.Mole()
mol.verbose = 5
mol.atom =''' 
  C   0.97091888623401     -0.07609978449644     -0.06440023695551
  N   -0.17562187186916     -0.07610010775030     -0.06439988152408
  H   2.03800298563516     -0.07610010775326     -0.06439988152041
 '''
mol.unit='A'
mol.basis = 'cc-pvtz'
#mol.cart=True
mol.build()




mf = dft.RKS(mol).density_fit()
mf.xc ='B3LYP'

mf.chkfile ='CHN.chk'
e_scf=mf.kernel()


title="CHN"
kpts=[]
from PyscfToQmcpack import savetoqmcpack
savetoqmcpack(mol,mf,title=title,kpts=kpts)

