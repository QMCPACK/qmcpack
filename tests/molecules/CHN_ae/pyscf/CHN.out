#INFO: **** input file is /home/abenali/Work/Applications/Anatol/QM9_AMONS_NI_LE_5/ALL_Runs/frag_00005/B3LYP/frag00005.py ****
#!/usr/bin/env python

from pyscf import gto
from pyscf import scf, dft, df

mol = gto.Mole()
mol.verbose = 5
mol.atom =''' 
  C   0.97091888623401     -0.07609978449644     -0.06440023695551
  N   -0.17562187186916     -0.07610010775030     -0.06439988152408
  H   2.03800298563516     -0.07610010775326     -0.06439988152041
 '''
mol.unit='A'
mol.basis = 'cc-pvtz'
#mol.cart=True
mol.build()




mf = dft.RKS(mol).density_fit()
mf.xc ='B3LYP'

mf.chkfile ='frag_00005.chk'
e_scf=mf.kernel()
#dm = mf.from_chk('C5H12-dft.chk')
#e_scf=mf.kernel(dm)


title="frag_00005"
kpts=[]
from PyscfToQmcpack import savetoqmcpack
savetoqmcpack(mol,mf,title=title,kpts=kpts)

#INFO: ******************** input file end ********************


System: ('Linux', 'abenali', '4.15.0-39-generic', '#42-Ubuntu SMP Tue Oct 23 15:48:01 UTC 2018', 'x86_64', 'x86_64')  Threads 8
Python 2.7.15rc1 (default, Nov 12 2018, 14:31:15) 
[GCC 7.3.0]
numpy 1.14.0  scipy 1.0.0
Date: Mon Nov 26 04:03:42 2018
PySCF version 1.6a
PySCF path  /home/abenali/Work/src/pyscf/pyscf
GIT ORIG_HEAD 7666303c9de36afb63ffe98cb0ed18d534b27cbc
GIT HEAD      ref: refs/heads/dev
GIT dev branch  2a6e84075bfe26ccd0bc363d0095d10c8db7b65f

[CONFIG] conf_file None
[INPUT] verbose = 5
[INPUT] max_memory = 4000 
[INPUT] num. atoms = 3
[INPUT] num. electrons = 14
[INPUT] charge = 0
[INPUT] spin (= nelec alpha-beta = 2S) = 0
[INPUT] symmetry False subgroup None
[INPUT] Mole.unit = A
[INPUT]  1 C      0.970918886234  -0.076099784496  -0.064400236956 AA    1.834770784150  -0.143807750837  -0.121698810203 Bohr
[INPUT]  2 N     -0.175621871869  -0.076100107750  -0.064399881524 AA   -0.331877239316  -0.143808361698  -0.121698138535 Bohr
[INPUT]  3 H      2.038002985635  -0.076100107753  -0.064399881520 AA    3.851267483896  -0.143808361704  -0.121698138528 Bohr
[INPUT] ---------------- BASIS SET ---------------- 
[INPUT] l, kappa, [nprim/nctr], expnt,             c_1 c_2 ...
[INPUT] H
[INPUT] 0    0    [3    /1   ]  33.87             0.006068
                                5.095             0.045308
                                1.159             0.202822
[INPUT] 0    0    [1    /1   ]  0.3258               1
[INPUT] 0    0    [1    /1   ]  0.1027               1
[INPUT] 1    0    [1    /1   ]  1.407                1
[INPUT] 1    0    [1    /1   ]  0.388                1
[INPUT] 2    0    [1    /1   ]  1.057                1
[INPUT] C
[INPUT] 0    0    [8    /2   ]  8236              0.000531 -0.000113
                                1235              0.004108 -0.000878
                                280.8             0.021087 -0.00454
                                79.27             0.081853 -0.018133
                                25.59             0.234817 -0.05576
                                8.997             0.434401 -0.126895
                                3.319             0.346129 -0.170352
                                0.3643            -0.008983 0.598684
[INPUT] 0    0    [1    /1   ]  0.9059               1
[INPUT] 0    0    [1    /1   ]  0.1285               1
[INPUT] 1    0    [3    /1   ]  18.71             0.014031
                                4.133             0.086866
                                1.2               0.290216
[INPUT] 1    0    [1    /1   ]  0.3827               1
[INPUT] 1    0    [1    /1   ]  0.1209               1
[INPUT] 2    0    [1    /1   ]  1.097                1
[INPUT] 2    0    [1    /1   ]  0.318                1
[INPUT] 3    0    [1    /1   ]  0.761                1
[INPUT] N
[INPUT] 0    0    [8    /2   ]  11420             0.000523 -0.000115
                                1712              0.004045 -0.000895
                                389.3             0.020775 -0.004624
                                110               0.080727 -0.018528
                                35.57             0.233074 -0.057339
                                12.54             0.433501 -0.132076
                                4.644             0.347472 -0.17251
                                0.5118            -0.008508 0.599944
[INPUT] 0    0    [1    /1   ]  1.293                1
[INPUT] 0    0    [1    /1   ]  0.1787               1
[INPUT] 1    0    [3    /1   ]  26.63             0.01467
                                5.948             0.091764
                                1.742             0.298683
[INPUT] 1    0    [1    /1   ]  0.555                1
[INPUT] 1    0    [1    /1   ]  0.1725               1
[INPUT] 2    0    [1    /1   ]  1.654                1
[INPUT] 2    0    [1    /1   ]  0.469                1
[INPUT] 3    0    [1    /1   ]  1.093                1

nuclear repulsion = 24.0336217876244
number of shells = 24
number of NR pGTOs = 100
number of NR cGTOs = 74
basis = cc-pvtz
ecp = {}
CPU time:         0.51


******** <class 'pyscf.df.df_jk.DFHF'> ********
method = DFHF-RKS-RHF
initial guess = minao
damping factor = 0
level shift factor = 0
DIIS = <class 'pyscf.scf.diis.CDIIS'>
DIIS start cycle = 1
DIIS space = 8
SCF tol = 1e-09
SCF gradient tol = None
max. SCF cycles = 50
direct_scf = False
chkfile to save SCF result = frag_00005.chk
max_memory 4000 MB (current use 62 MB)
XC functionals = B3LYP
small_rho_cutoff = 1e-07
radial grids: 
    Treutler-Ahlrichs (JCP 102, 346 (M4)) radial grids
    
becke partition: Becke, JCP, 88, 2547 (1988)
pruning grids: <function nwchem_prune at 0x14cea412c578>
grids dens level: 3
symmetrized grids: False
atomic radii adjust function: <function treutler_atomic_radii_adjust at 0x14cea412c488>
Set gradient conv threshold to 3.16228e-05
cond(S) = 4498.290336324065
atom C rad-grids = 75, ang-grids = [ 50  50  50  50  50  50  50  50  50  50  50  50  50  50  50  50  50  50
  50  50  50  50  50  86  86  86  86  86  86  86  86  86  86 266 266 266
 266 266 266 266 302 302 302 302 302 302 302 302 302 302 302 302 302 302
 302 302 302 302 302 302 266 266 266 266 266 266 266 266 266 266 266 266
 266 266 266]
atom N rad-grids = 75, ang-grids = [ 50  50  50  50  50  50  50  50  50  50  50  50  50  50  50  50  50  50
  50  50  50  50  50  86  86  86  86  86  86  86  86  86 266 266 266 266
 266 266 266 302 302 302 302 302 302 302 302 302 302 302 302 302 302 302
 302 302 302 302 266 266 266 266 266 266 266 266 266 266 266 266 266 266
 266 266 266]
atom H rad-grids = 50, ang-grids = [ 50  50  50  50  50  50  50  50  50  50  50  50  50  50  86  86  86  86
 266 266 266 266 302 302 302 302 302 302 302 302 302 302 302 302 302 266
 266 266 266 266 266 266 266 266 266 266 266 266 266 266]
tot grids = 37972
Drop grids 4185
    CPU time for setting up grids      0.88 sec, wall time      0.11 sec
nelec by numeric integration = 13.998750653981666
    CPU time for vxc      1.58 sec, wall time      0.20 sec
******** <class 'pyscf.df.df.DF'> ********
auxbasis = None
max_memory = 4000
_cderi_to_save = /home/abenali/Work/Applications/Anatol/QM9_AMONS_NI_LE_5/ALL_Runs/frag_00005/B3LYP/tmpBNIQfI
Default auxbasis cc-pvtz-jkfit is used for H cc-pvtz
Default auxbasis cc-pvtz-jkfit is used for C cc-pvtz
Default auxbasis cc-pvtz-jkfit is used for N cc-pvtz
num shells = 60, num cGTOs = 188
size of aux basis 188
    CPU time for 2c2e      0.00 sec, wall time      0.00 sec
    CPU time for 3c2e      0.03 sec, wall time      0.00 sec
    CPU time for Cholesky 2c2e      0.07 sec, wall time      0.01 sec
    CPU time for cholesky_eri      0.18 sec, wall time      0.02 sec
    CPU time for vj and vk      0.74 sec, wall time      0.09 sec
E1 = -172.06205058908034  Ecoul = 66.92968387831496  Exc = -12.418009180206269
init E= -93.5167541033473
    CPU time for initialize scf      3.30 sec, wall time      0.42 sec
  HOMO = -0.415040229563477  LUMO = -0.0600361939875198
  mo_energy =
[-14.41134967 -10.43728954  -1.08176293  -0.63152995  -0.4356593
  -0.41504023  -0.41504023  -0.06003619  -0.06003619   0.04105251
   0.10690601   0.26402788   0.26402788   0.31555395   0.38564726
   0.45208779   0.48095415   0.48095415   0.60541757   0.60541758
   0.68158176   0.68158176   0.72267992   0.9139549    1.13611681
   1.13611681   1.15002776   1.15002776   1.37266487   1.46642199
   1.61817541   1.61817541   1.71803273   1.90618384   1.90618384
   2.14762682   2.47243619   2.58430691   2.58430708   2.5940557
   2.5940557    2.66720227   2.66720227   2.83731872   2.83731937
   2.98372538   2.98372538   3.05981319   3.16573042   3.16573042
   3.52480396   3.52480495   3.58247473   3.88893112   3.88893112
   3.99679956   4.00671848   4.00671848   4.42390932   4.42391339
   4.51308837   4.51308837   4.60433046   4.60433166   4.63319134
   5.00436538   5.21660953   5.21660953   5.60862733   6.1616108
   6.1616108    6.32290774   8.12412482  14.52297178]
nelec by numeric integration = 14.000000012194187
    CPU time for vxc      1.51 sec, wall time      0.19 sec
    CPU time for vj and vk      0.16 sec, wall time      0.02 sec
E1 = -174.1950322820547  Ecoul = 69.4648008969596  Exc = -12.68765591952379
cycle= 1 E= -93.3842655169945  delta_E= 0.132  |g|= 0.318  |ddm|= 1.48
    CPU time for cycle= 1      1.86 sec, wall time      0.23 sec
  HOMO = -0.303350706418472  LUMO = 0.0334925168709631
  mo_energy =
[-14.16934224 -10.13221786  -0.83630013  -0.61102466  -0.3062961
  -0.3062961   -0.30335071   0.03349252   0.0489912    0.0489912
   0.13331414   0.30982224   0.30982224   0.35569091   0.41168154
   0.50011235   0.56641174   0.56641174   0.65233153   0.65233153
   0.67981241   0.67981241   0.79805433   1.02823145   1.19070245
   1.19070245   1.23532592   1.23532593   1.40239496   1.53244073
   1.71590272   1.71590272   1.84978562   2.03264473   2.03264473
   2.18465773   2.57206181   2.6483115    2.64831174   2.66622762
   2.66622762   2.73072495   2.73072495   2.93823439   2.93823488
   3.13107344   3.1358329    3.1358329    3.26281776   3.26281776
   3.53194556   3.53194655   3.73122426   3.99642653   3.99642653
   4.03826698   4.03826698   4.05221187   4.60503867   4.60504254
   4.62291317   4.62291317   4.72364034   4.74107416   4.74107571
   5.1145161    5.33882455   5.33882455   5.77300406   6.29165719
   6.29165719   6.49217969   8.21684011  14.70135388]
nelec by numeric integration = 14.000000011798999
    CPU time for vxc      1.72 sec, wall time      0.21 sec
    CPU time for vj and vk      0.15 sec, wall time      0.02 sec
E1 = -171.82990299667878  Ecoul = 66.84163802931457  Exc = -12.403397268253247
cycle= 2 E= -93.3580404479931  delta_E= 0.0262  |g|= 0.473  |ddm|= 0.538
    CPU time for cycle= 2      2.18 sec, wall time      0.27 sec
  HOMO = -0.359625394031787  LUMO = 0.0134922217131153
  mo_energy =
[-1.43605056e+01 -1.01649811e+01 -9.15788434e-01 -5.97146315e-01
 -3.88950608e-01 -3.59625394e-01 -3.59625394e-01  1.34922217e-02
  1.34922217e-02  4.61998853e-02  1.24978540e-01  3.01456262e-01
  3.01456262e-01  3.40216646e-01  4.15878236e-01  4.79249129e-01
  5.22373674e-01  5.22373674e-01  6.40650881e-01  6.40650882e-01
  6.87525373e-01  6.87525373e-01  7.76498030e-01  9.75391032e-01
  1.17416824e+00  1.17416824e+00  1.18507602e+00  1.18507603e+00
  1.40142519e+00  1.50993327e+00  1.67110715e+00  1.67110715e+00
  1.79896020e+00  2.01361413e+00  2.01361413e+00  2.19028823e+00
  2.54064470e+00  2.63981239e+00  2.63981259e+00  2.65608554e+00
  2.65608554e+00  2.73397694e+00  2.73397694e+00  2.92686007e+00
  2.92686058e+00  3.05709764e+00  3.05709764e+00  3.11814560e+00
  3.22082117e+00  3.22082117e+00  3.53846355e+00  3.53846453e+00
  3.65759471e+00  3.95412548e+00  3.95412548e+00  4.00527871e+00
  4.00527871e+00  4.03118383e+00  4.49837287e+00  4.49837715e+00
  4.57358179e+00  4.57358179e+00  4.67200055e+00  4.67200171e+00
  4.69648256e+00  5.06667969e+00  5.28422683e+00  5.28422683e+00
  5.71199766e+00  6.23885525e+00  6.23885525e+00  6.43617562e+00
  8.19214805e+00  1.46230838e+01]
nelec by numeric integration = 14.000000016506116
    CPU time for vxc      1.50 sec, wall time      0.19 sec
    CPU time for vj and vk      0.15 sec, wall time      0.02 sec
E1 = -173.11395045476723  Ecoul = 68.24006639679317  Exc = -12.562298956918669
cycle= 3 E= -93.4025612272684  delta_E= -0.0445  |g|= 0.185  |ddm|= 0.352
    CPU time for cycle= 3      1.72 sec, wall time      0.22 sec
  HOMO = -0.370597003595538  LUMO = 0.0047779397678662
  mo_energy =
[-1.43454476e+01 -1.02324536e+01 -9.30759244e-01 -6.14286777e-01
 -3.87020212e-01 -3.70597004e-01 -3.70597004e-01  4.77793977e-03
  4.77793977e-03  4.04146103e-02  1.20766085e-01  2.89094534e-01
  2.89094534e-01  3.39493077e-01  4.07512895e-01  4.76111379e-01
  5.22526672e-01  5.22526672e-01  6.28602205e-01  6.28602208e-01
  6.80885825e-01  6.80885825e-01  7.65992875e-01  9.72112466e-01
  1.16836856e+00  1.16836856e+00  1.18406679e+00  1.18406680e+00
  1.39185738e+00  1.50260870e+00  1.66908372e+00  1.66908372e+00
  1.78904513e+00  1.98333043e+00  1.98333043e+00  2.17741345e+00
  2.52813651e+00  2.62444352e+00  2.62444371e+00  2.63464959e+00
  2.63464959e+00  2.71330231e+00  2.71330231e+00  2.89824799e+00
  2.89824854e+00  3.05282858e+00  3.05282858e+00  3.10533726e+00
  3.21394152e+00  3.21394152e+00  3.52974821e+00  3.52974920e+00
  3.65452362e+00  3.95596972e+00  3.95596972e+00  4.00109728e+00
  4.00109728e+00  4.02513319e+00  4.50126662e+00  4.50127077e+00
  4.57015809e+00  4.57015809e+00  4.67027031e+00  4.67027159e+00
  4.68555544e+00  5.06439147e+00  5.27924582e+00  5.27924582e+00
  5.69734662e+00  6.22935754e+00  6.22935754e+00  6.41920649e+00
  8.17595915e+00  1.46084226e+01]
nelec by numeric integration = 14.000000016026842
    CPU time for vxc      1.50 sec, wall time      0.19 sec
    CPU time for vj and vk      0.15 sec, wall time      0.02 sec
E1 = -173.15500460886463  Ecoul = 68.27295530864595  Exc = -12.561185135833298
cycle= 4 E= -93.4096126484276  delta_E= -0.00705  |g|= 0.0164  |ddm|= 0.0797
    CPU time for cycle= 4      1.72 sec, wall time      0.22 sec
  HOMO = -0.368281682199707  LUMO = 0.00715795868303211
  mo_energy =
[-1.43396556e+01 -1.02272030e+01 -9.27439456e-01 -6.10017239e-01
 -3.86870776e-01 -3.68281682e-01 -3.68281682e-01  7.15795868e-03
  7.15795868e-03  4.27093325e-02  1.22033836e-01  2.91034394e-01
  2.91034394e-01  3.40114665e-01  4.09686354e-01  4.76526186e-01
  5.23057553e-01  5.23057553e-01  6.30906166e-01  6.30906170e-01
  6.83284635e-01  6.83284635e-01  7.68825980e-01  9.72895498e-01
  1.17036208e+00  1.17036208e+00  1.18463214e+00  1.18463215e+00
  1.39415654e+00  1.50484180e+00  1.66962374e+00  1.66962374e+00
  1.79090480e+00  1.98702002e+00  1.98702002e+00  2.18096797e+00
  2.53060267e+00  2.62786815e+00  2.62786835e+00  2.63816394e+00
  2.63816394e+00  2.71757810e+00  2.71757810e+00  2.90252578e+00
  2.90252632e+00  3.05570743e+00  3.05570743e+00  3.10922033e+00
  3.21671828e+00  3.21671828e+00  3.53313891e+00  3.53313990e+00
  3.65622167e+00  3.95664482e+00  3.95664482e+00  4.00431679e+00
  4.00431679e+00  4.02832341e+00  4.50291192e+00  4.50291607e+00
  4.57251124e+00  4.57251124e+00  4.67133458e+00  4.67133584e+00
  4.68905485e+00  5.06607449e+00  5.28119537e+00  5.28119537e+00
  5.70086493e+00  6.23222152e+00  6.23222152e+00  6.42314034e+00
  8.17945225e+00  1.46124094e+01]
nelec by numeric integration = 14.000000015942941
    CPU time for vxc      1.48 sec, wall time      0.19 sec
    CPU time for vj and vk      0.18 sec, wall time      0.02 sec
E1 = -173.13032998524343  Ecoul = 68.24515056144372  Exc = -12.558119301588041
cycle= 5 E= -93.4096769377634  delta_E= -6.43e-05  |g|= 0.00142  |ddm|= 0.00892
    CPU time for cycle= 5      1.74 sec, wall time      0.22 sec
  HOMO = -0.368490850603924  LUMO = 0.00694855675130921
  mo_energy =
[-1.43407576e+01 -1.02273186e+01 -9.27702547e-01 -6.10560035e-01
 -3.87011869e-01 -3.68490851e-01 -3.68490851e-01  6.94855675e-03
  6.94855675e-03  4.24173200e-02  1.21911016e-01  2.90927470e-01
  2.90927470e-01  3.39994327e-01  4.09467858e-01  4.76495877e-01
  5.22985631e-01  5.22985631e-01  6.30772776e-01  6.30772779e-01
  6.82952409e-01  6.82952409e-01  7.68498211e-01  9.72827844e-01
  1.17013954e+00  1.17013954e+00  1.18459016e+00  1.18459017e+00
  1.39388859e+00  1.50454243e+00  1.66955097e+00  1.66955097e+00
  1.79074779e+00  1.98685556e+00  1.98685556e+00  2.18053927e+00
  2.53037529e+00  2.62754579e+00  2.62754599e+00  2.63798039e+00
  2.63798039e+00  2.71714587e+00  2.71714587e+00  2.90228455e+00
  2.90228510e+00  3.05535166e+00  3.05535166e+00  3.10871553e+00
  3.21634986e+00  3.21634986e+00  3.53265537e+00  3.53265636e+00
  3.65597355e+00  3.95652378e+00  3.95652378e+00  4.00375996e+00
  4.00375996e+00  4.02785021e+00  4.50260186e+00  4.50260602e+00
  4.57212784e+00  4.57212784e+00  4.67118633e+00  4.67118759e+00
  4.68855795e+00  5.06581253e+00  5.28093459e+00  5.28093459e+00
  5.70047017e+00  6.23188803e+00  6.23188803e+00  6.42272683e+00
  8.17905672e+00  1.46119528e+01]
nelec by numeric integration = 14.00000001598467
    CPU time for vxc      1.45 sec, wall time      0.24 sec
    CPU time for vj and vk      0.28 sec, wall time      0.04 sec
E1 = -173.13342117616892  Ecoul = 68.24874398446208  Exc = -12.55862189958619
cycle= 6 E= -93.4096773036686  delta_E= -3.66e-07  |g|= 9.83e-05  |ddm|= 0.000922
    CPU time for cycle= 6      1.81 sec, wall time      0.29 sec
  HOMO = -0.368483312574009  LUMO = 0.00695930965862389
  mo_energy =
[-1.43407494e+01 -1.02272301e+01 -9.27695333e-01 -6.10537167e-01
 -3.86987882e-01 -3.68483313e-01 -3.68483313e-01  6.95930966e-03
  6.95930966e-03  4.24252694e-02  1.21911451e-01  2.90931078e-01
  2.90931078e-01  3.40003272e-01  4.09478004e-01  4.76504640e-01
  5.22991756e-01  5.22991756e-01  6.30769632e-01  6.30769635e-01
  6.82959299e-01  6.82959299e-01  7.68507699e-01  9.72840330e-01
  1.17014372e+00  1.17014372e+00  1.18459138e+00  1.18459139e+00
  1.39390318e+00  1.50455824e+00  1.66956565e+00  1.66956565e+00
  1.79076109e+00  1.98687513e+00  1.98687513e+00  2.18055422e+00
  2.53039269e+00  2.62755323e+00  2.62755343e+00  2.63798182e+00
  2.63798182e+00  2.71716494e+00  2.71716494e+00  2.90229687e+00
  2.90229742e+00  3.05536192e+00  3.05536192e+00  3.10873869e+00
  3.21635872e+00  3.21635872e+00  3.53266522e+00  3.53266621e+00
  3.65599177e+00  3.95652374e+00  3.95652374e+00  4.00377274e+00
  4.00377274e+00  4.02786674e+00  4.50260830e+00  4.50261246e+00
  4.57214937e+00  4.57214937e+00  4.67119889e+00  4.67120015e+00
  4.68858497e+00  5.06583854e+00  5.28095662e+00  5.28095662e+00
  5.70049202e+00  6.23190179e+00  6.23190179e+00  6.42275320e+00
  8.17908193e+00  1.46119766e+01]
nelec by numeric integration = 14.000000015986371
    CPU time for vxc      1.54 sec, wall time      0.21 sec
    CPU time for vj and vk      0.25 sec, wall time      0.03 sec
E1 = -173.133220270259  Ecoul = 68.2485094012452  Exc = -12.558588224040896
cycle= 7 E= -93.4096773054303  delta_E= -1.76e-09  |g|= 2.4e-05  |ddm|= 7.36e-05
    CPU time for cycle= 7      1.89 sec, wall time      0.25 sec
  HOMO = -0.36848642006067  LUMO = 0.00695567277151708
  mo_energy =
[-1.43407628e+01 -1.02272368e+01 -9.27698007e-01 -6.10540382e-01
 -3.86994205e-01 -3.68486420e-01 -3.68486420e-01  6.95567277e-03
  6.95567277e-03  4.24244092e-02  1.21910357e-01  2.90929661e-01
  2.90929661e-01  3.40001152e-01  4.09476657e-01  4.76503113e-01
  5.22989227e-01  5.22989227e-01  6.30767981e-01  6.30767984e-01
  6.82957600e-01  6.82957600e-01  7.68505364e-01  9.72836785e-01
  1.17014145e+00  1.17014145e+00  1.18458812e+00  1.18458813e+00
  1.39390098e+00  1.50455553e+00  1.66956231e+00  1.66956231e+00
  1.79075796e+00  1.98687199e+00  1.98687199e+00  2.18055166e+00
  2.53038965e+00  2.62755071e+00  2.62755091e+00  2.63797965e+00
  2.63797965e+00  2.71716163e+00  2.71716163e+00  2.90229384e+00
  2.90229439e+00  3.05535675e+00  3.05535675e+00  3.10873480e+00
  3.21635543e+00  3.21635543e+00  3.53266285e+00  3.53266384e+00
  3.65598664e+00  3.95651883e+00  3.95651883e+00  4.00377056e+00
  4.00377056e+00  4.02786349e+00  4.50260285e+00  4.50260701e+00
  4.57214431e+00  4.57214431e+00  4.67119375e+00  4.67119501e+00
  4.68858034e+00  5.06583357e+00  5.28095160e+00  5.28095160e+00
  5.70048812e+00  6.23189822e+00  6.23189822e+00  6.42274920e+00
  8.17907744e+00  1.46119705e+01]
nelec by numeric integration = 14.000000015986517
    CPU time for vxc      1.58 sec, wall time      0.20 sec
    CPU time for vj and vk      0.18 sec, wall time      0.02 sec
E1 = -173.13327792561992  Ecoul = 68.24857539535802  Exc = -12.558596562906835
cycle= 8 E= -93.4096773055444  delta_E= -1.14e-10  |g|= 2.26e-06  |ddm|= 2.15e-05
    CPU time for cycle= 8      1.87 sec, wall time      0.23 sec
  HOMO = -0.368485484221655  LUMO = 0.00695644980693009
  mo_energy =
[-1.43407608e+01 -1.02272348e+01 -9.27696996e-01 -6.10539598e-01
 -3.86993513e-01 -3.68485484e-01 -3.68485484e-01  6.95644981e-03
  6.95644981e-03  4.24246154e-02  1.21910668e-01  2.90930243e-01
  2.90930243e-01  3.40001360e-01  4.09477056e-01  4.76503390e-01
  5.22989621e-01  5.22989621e-01  6.30768675e-01  6.30768678e-01
  6.82957922e-01  6.82957922e-01  7.68505993e-01  9.72837349e-01
  1.17014188e+00  1.17014188e+00  1.18458858e+00  1.18458859e+00
  1.39390147e+00  1.50455599e+00  1.66956278e+00  1.66956278e+00
  1.79075871e+00  1.98687323e+00  1.98687323e+00  2.18055219e+00
  2.53039031e+00  2.62755147e+00  2.62755167e+00  2.63798074e+00
  2.63798074e+00  2.71716260e+00  2.71716260e+00  2.90229512e+00
  2.90229567e+00  3.05535773e+00  3.05535773e+00  3.10873552e+00
  3.21635599e+00  3.21635599e+00  3.53266321e+00  3.53266420e+00
  3.65598746e+00  3.95651942e+00  3.95651942e+00  4.00377067e+00
  4.00377067e+00  4.02786395e+00  4.50260371e+00  4.50260787e+00
  4.57214500e+00  4.57214500e+00  4.67119444e+00  4.67119570e+00
  4.68858109e+00  5.06583421e+00  5.28095236e+00  5.28095236e+00
  5.70048907e+00  6.23189900e+00  6.23189900e+00  6.42275017e+00
  8.17907830e+00  1.46119718e+01]
nelec by numeric integration = 14.00000001598645
    CPU time for vxc      1.58 sec, wall time      0.20 sec
    CPU time for vj and vk      0.15 sec, wall time      0.02 sec
E1 = -173.1332663055359  Ecoul = 68.2485623435625  Exc = -12.558595131195744
Extra cycle  E= -93.4096773055448  delta_E= -3.98e-13  |g|= 1.82e-06  |ddm|= 2.68e-06
    CPU time for scf_cycle     19.89 sec, wall time      2.58 sec
    CPU time for SCF     19.89 sec, wall time      2.58 sec
converged SCF energy = -93.4096773055448
Using Python 2.x
Wavefunction successfully saved to QMCPACK HDF5 Format
Use: "convert4qmc -pyscf  frag_00005.h5" to generate QMCPACK input files
